src/components/course/CourseSidebarPrice.jsx

```diff
@@ -64,9 +72,28 @@ const CourseSidebarPrice = () => {
 
   // Case 3: No subsidies found
   if (!hasDiscountedPrice) {
-    return <span>${originalPriceDisplay} {currency}</span>;
+    const { catalogList } = courseData.catalog;
+    const hasOfferWithInsufficientBalance = canEnrollWithEnterpriseOffers && enterpriseOffers.find(
+      (enterpriseOffer) => catalogList?.includes(enterpriseOffer.enterpriseCatalogUuid)
+       && enterpriseOffer.remainingBalance < coursePrice.list,
+    );
+
+    return (
+      <>
+        <span className="d-block">${originalPriceDisplay} {currency}</span>
+        {hasOfferWithInsufficientBalance && (
+          <small data-testid="insufficient-offer-balance-text">
+            Your organization doesn&apos;t have sufficient balance to cover this course.{' '}
```

Comment:
nit: Should this text also be a constant, as it seems that's the convention (no strong opinions haha)?

Reply:
Oh, should this also say "learner credit balance" to be consistent with the other message?